example/lib/example_peripheral.dart

```diff
@@ -98,6 +98,16 @@ class TemperatureService extends SimulatedService {
             convenienceName: convenienceName) {
     characteristicByUuid(_temperatureConfigUuid).monitor().listen((value) {
       _readingTemperature = value[0] == 1;
+
+      if (_readingTemperature) {
+        SimulatedCharacteristic temperatureDataCharacteristic =
+            characteristicByUuid(_temperatureDataUuid);
+
+        temperatureDataCharacteristic.write(
+          Uint8List.fromList([0, 0, 100, Random().nextInt(255)]),
+          sendNotification: false,
```

Comment:
I know it's only an example but why you disabled notifying about temperature changes?

Reply:
The notifications about the temperature are decoupled from actual reading of the sensor. I mean: there can be a value to be read instantly after turning on the sensor, but you will be notified only after the set interval.